Lesson_9/Tusk_1.py

```diff
@@ -0,0 +1,24 @@
+def fib_gen(target_number):
+    num1 = 0
+    num2 = 1
+    count = 1
+
+    while count < target_number:
+        num3 = num1 + num2
+        num1 = num2
+        num2 = num3
+        count += 1
+        yield num2, count
+
+
+target = int(input('Which number from Fibonacci Sequence do you want to see? '))
+
+if target == 0:
+    print('Requested number is 0')
+elif target == 1:
+    print('Requested number is 1')
+else:
+    for item in fib_gen(target):
```

Comment:
To get the last value of the generated collection (and to change the code according to my previous comment), you can transform the generator to a list and get its last value: `list(fib_gen(target))[-1]`

Reply:
I did the update, Thank you for advice.